src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -616,6 +620,10 @@ namespace xclhwemhal2 {
         setenv("VITIS_KERNEL_PROFILE_FILENAME", kernelProfileFileName.c_str(), true);
         setenv("VITIS_KERNEL_TRACE_FILENAME", kernelTraceFileName.c_str(), true);
       }
+      if (lWaveform == xclemulation::LAUNCHWAVEFORM::OFF)
+      {
+        sim_path = binaryDirectory + "/behav_gdb/xsim";
+      }
```

Comment:
Please be consistent with the use of curly braces after conditionals.    The convention followed by 'most' XRT code and also used in some places in this PR,  is open curly brace on same line 'if'.  No curly braces for single statement, instead a new line after the single statement.

Reply:
As suggested, I have removed the curly braces for single line if statements.Please review.